embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/arch/destination/SessionSpanWriter.kt

```diff
@@ -21,15 +21,11 @@ public interface SessionSpanWriter {
 
     /**
      * Add the given key-value pair as an Attribute to the session span
-     *
-     * Returns true if the attribute was added, otherwise false.
      */
-    public fun addCustomAttribute(attribute: SpanAttributeData): Boolean
+    public fun addSystemAttribute(attribute: SpanAttributeData)
```

Comment:
This always succeeds now because we are adding to a map without additional validation.